web/src/util.ts

```diff
@@ -53,14 +53,9 @@ export const lower_same = function lower_same(a?: string, b?: string): boolean {
     return a.toLowerCase() === b.toLowerCase();
 };
 
-export type StreamTopic = {
-    stream_id: number;
-    topic: string;
-};
-
 export const same_stream_and_topic = function util_same_stream_and_topic(
-    a: StreamTopic,
-    b: StreamTopic,
+    a: {stream_id?: number; topic: string},
+    b: {stream_id?: number; topic: string},
 ): boolean {
```

Comment:
I would prefer to fix the caller to not use this function when it's not a known stream message, over making this function not insist on expected types.

Reply:
Sure, removing this and tweaking the callsite.